cql3/expr/expression.cc

```diff
@@ -549,12 +550,14 @@ bool is_satisfied_by(const expression& restr, const evaluation_inputs& inputs) {
 
 namespace {
 
-template<typename Range>
-value_list to_sorted_vector(Range r, const serialized_compare& comparator) {
-    BOOST_CONCEPT_ASSERT((boost::ForwardRangeConcept<Range>));
-    value_list tmp(r.begin(), r.end()); // Need random-access range to sort (r is not necessarily random-access).
-    const auto unique = boost::unique(boost::sort(tmp, comparator));
-    return value_list(unique.begin(), unique.end());
+template<std::ranges::range Range>
+value_list to_sorted_vector(Range&& r, const serialized_compare& comparator) {
+    value_list unique; // Need random-access range to sort (r is not necessarily random-access).
+    std::ranges::move(r.begin(), r.end(), std::back_inserter(unique));
```

Comment:
Actually, it's a static template in this file, called with either `utils::chunked_vector<managed_bytes_opt>` or `std::vector<managed_bytes_opt>` so they can be moved.
But I agree it can be seen as out of scope.
I can send it as a followup patch